tests/behaviour/behaviour.test.ts

```diff
@@ -33,10 +33,11 @@ interface AsyncTestCluster {
   dependencies: Map<string, string[]>;
 }
 
-fs.writeFileSync(
-  path.join(BASE_PATH, '.starknet_accounts_devnet', 'starknet_open_zeppelin_accounts.json'),
-  '{}',
-);
+const accountDir = '.starknet_accounts_devnet';
+if (!fs.existsSync(accountDir)) {
+  fs.mkdirSync(accountDir);
+}
```

Comment:
```suggestion
fs.mkdirSync(accountDir, { recursive: true })
```

Reply:
It's only one folder to create, seems better to use a more restrict approach

Reply:
Unfortunately, FS is a naturally parallelly-accessed system, so checks like that are prone to data races. It might not be to much of a concern in CI at this moment, but why pessimize)